Mollie/API/Resource/Customers.py

```diff
@@ -0,0 +1,26 @@
+from .Base import *
+from Mollie.API.Error import *
+from Mollie.API.Object import Customer
+
+
+class Customers(Base):
+    RESOURCE_ID_PREFIX = 'cst_'
+
+    def getResourceObject(self, result):
+        return Customer(result)
+
+    def get(self, customer_id):
+        if not customer_id or self.RESOURCE_ID_PREFIX not in customer_id:
```

Comment:
It would be neater to actually check that the customer_id starts with the prefix, not only that it contains it.